win32/appveyor.bat

```diff
@@ -86,22 +86,22 @@ copy %ICONV_BUILD_DIR%\msvc10\iconv.dll %APPVEYOR_BUILD_FOLDER% > nul
 cd %APPVEYOR_BUILD_FOLDER%
 nmake -f mk_mvc.mak WITH_ICONV=yes ICONV_DIR=%ICONV_DIR% PDB=yes || exit 1
 
-:: Backup VC binaries
-mkdir vc
-move *.exe vc > nul
+rem :: Backup VC binaries

```

Comment:
In this commit, you converted some lines to comments with "rem".
I wonder why these conversions are needed.

Reply:
Before this commit, we generated a Makefile by MSYS2 and used it for testing. When generating the Makefile, MSVC binaries were overwritten by MSYS2, so we needed to backup them.
Now I stop generating the Makefile, and use misc/units.py directly instead. This reduces the testing time about 2 minutes (or more).

`rem` is a comment. I can also remove those lines.

Reply:
I see.
As you proposed, I would like you to remove the comment lines. git may help us when we want to go back to the original code.

Reply:
Okay, updated.